.github/pull_request_template.md

```diff
@@ -11,8 +11,8 @@ As the author of this PR, please check off the items in this checklist:
 - [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
 - [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
 - [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
-- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
-  functionality, content, code)
+- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
+- [ ] Has updated the documentation of minimum requirements if any have changed, such as documenting K8s version bumps
```

Comment:
I think this is redundant with the first bullet, and want to avoid adding too many things to the contributor checklist, but will leave up to you

Reply:
Well, I think it needs to be highlighted because it keeps on being missed, can add it as a sub-bullet of the first one

Reply:
sg!